app/src/org/commcare/engine/resource/AndroidResourceManager.java

```diff
@@ -141,7 +143,7 @@ private void loadProfileViaTemp(Resource upgradeProfile)
             UnresolvedResourceException,
             InstallCancelledException {
         tempUpgradeTable.destroy();
-        loadProfileIntoTable(tempUpgradeTable, profileRef);
+        loadRemoteAuthorityProfileIntoTable(tempUpgradeTable, profileRef);
```

Comment:
good call, fixed. I actually just got rid of the `loadRemoteAuthorityProfileIntoTable` method entirely since it was only being used in 1 other place.